torch/fx/examples/replace_op.py

```diff
@@ -0,0 +1,64 @@
+import torch
+from torch.fx import symbolic_trace
+import operator
+
+
+"""
+How to replace one op with another
+1. Iterate through all nodes in your GraphModule's Graph.
+2. Determine if the current node should be replaced. (Suggested: match
+on the node's ``target`` attribute).
+3. Create a replacement node.
+4. Use the fx built-in ``replace_all_uses_with`` to replace the current
+node with the replacement.
+5. Call ``recompile`` on the GraphModule. This updates the generated
+Python code to reflect the new Graph state.
+
+Currently, FX does not provide operator normalization. It's up to the
+user to confirm a way to confirm that any replacement operators will
```

Comment:
`to confirm` seems to show up twice in this sentence. Also, a user reading this is probably not going to know what "operator normalization" means, so probably better to add a clause or example explaining the implication